src/s1reader/s1_reader.py

```diff
@@ -438,13 +439,14 @@ def load_bursts(path: str, orbit_path: str, swath_num: int, pol: str = 'vv'):
         Integer of subswath of desired burst. {1, 2, 3}
     pol : str
         Polarization of desired burst. {hh, vv, hv, vh}
+    burst_ids : list[str]
+        List of burst IDs to be returned. Default of None returns all bursts.
```

Comment:
Docstring is a bit confusing. It is the `Sentinel1BurstSlc` object for the specified burst ID that is returned (not really the burst ID XD)

Reply:
I agree. How about: "List of burst IDs for which their Sentinel1BurstSlc objects will be returned. Default of None returns all bursts."